docs/api/createAsyncThunk.mdx

```diff
@@ -98,10 +98,12 @@ An object with the following optional fields:
 
 - `condition(arg, { getState, extra } ): boolean`: a callback that can be used to skip execution of the payload creator and all action dispatches, if desired. See [Canceling Before Execution](#canceling-before-execution) for a complete description.
 - `dispatchConditionRejection`: if `condition()` returns `false`, the default behavior is that no actions will be dispatched at all. If you still want a "rejected" action to be dispatched when the thunk was canceled, set this flag to `true`.
-- `idGenerator(): string`: a function to use when generating the `requestId` for the request sequence. Defaults to use [nanoid](./otherExports.mdx/#nanoid).
+- `idGenerator(arg): string`: a function to use when generating the `requestId` for the request sequence. Defaults to use [nanoid](./otherExports.mdx/#nanoid), but you can implement your own ID generation logic.
 - `serializeError(error: unknown) => any` to replace the internal `miniSerializeError` method with your own serialization logic.
 - `getPendingMeta({ arg, requestId }, { getState, extra }): any`: a function to create an object that will be merged into the `pendingAction.meta` field.
 
+The options object gives you even more control over the async thunk internals. For example, you may customize how are requestIds generated by overriding the idGenerator option.
+
```

Comment:
Yeah, i did not notice how verbose this was, because i thought it would be helpful to explain the options object further, thus, encourage newcomers to experiment and fine tune the async thunk behavior. But yeah.

Reply:
Yeah, but I think then we would have to add all the options there, which would kinda repeat the paragraph above :smile: